src/scss/Landing.scss

```diff
@@ -39,37 +53,37 @@ $mobile: 500px;
           display: none;
         }
       }
-      .mobile-buttons {
-        display: none;
-        @media screen and (max-width: $mobile) {
-          display: flex;
-          flex-direction: column;
-          width: 100%;
-          margin: 1.5rem 1rem 0.5rem 0.5rem;
-          line-height: 100%;
-        }
-        .login {
-          color: #e8ebef;
-          width: 180px;
-          height: 60px;
-          font-size: 20px;
-          background: none;
-          border: 1px solid #979797;
-          border-radius: 5px;
-          cursor: pointer;
-        }
-
-        .join {
-          font-size: 20px;
-          color: #e8ebef;
-          width: 180px;
-          height: 60px;
-          background: #2d90f5;
-          border: 1px solid #2d90f5;
-          border-radius: 5px;
-          cursor: pointer;
-        }
-      }
+      // .mobile-buttons {
```

Comment:
I personally would go even further here... just rip it out. It's still gonna be there in version history if we need to recover it. And it looks stable on https://dev.rxid.net